firmware/configuration/t88_default/bootloaderconfig.h

```diff
@@ -24,16 +23,16 @@
 /* ------------------------------------------------------------------------- */
 
 #define USB_CFG_IOPORTNAME      D
-  /* This is the port where the USB bus is connected. When you configure it to
-   * "B", the registers PORTB, PINB and DDRB will be used.
-   */
+/* This is the port where the USB bus is connected. When you configure it to
+ * "B", the registers PORTB, PINB and DDRB will be used.
+ */
 
-#define USB_CFG_DMINUS_BIT      2
```

Comment:
why is it changed?

Reply:
Armin	10.03.2021	Swapped D+ and D- for t88 to support MH-ET LIVE Tiny88 boards	#define USB_CFG_DMINUS_BIT      1**